### ISSUE TITLE
[Bug]: Changes are not showing

### ISSUE THREAD

**Issue by anakinsolo** [Shopify/cli#1924]
### Please confirm that you have:

- [X] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [X] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Theme

### Expected behavior

I was following this https://shopify.dev/docs/themes/getting-started/create. I had
- installed shopify cli locally
- had the correct development environment
- had the local preview store up and running after using this command `shopify theme dev --store=https://mystore`
- gone and changed the value of the title in the sections/main-404.liquid

I expected the title of that page to change

### Actual behavior

Nothing changed.

### Verbose output

```shell
* Syncing theme #148991705363 on felia-demo-1.myshopify.com
                                                                                                                                                                               100%
sections/featured-collection.liquid: Invalid schema: setting color_scheme type is invalid.
sections/featured-product.liquid: Invalid schema: setting color_scheme type is invalid.
sections/announcement-bar.liquid: Invalid schema: setting color_scheme type is invalid.
sections/collage.liquid: Invalid schema: setting color_scheme type is invalid.
sections/collapsible-content.liquid: Invalid schema: setting color_scheme type is invalid.
sections/collection-list.liquid: Invalid schema: setting color_scheme type is invalid.
sections/contact-form.liquid: Invalid schema: setting color_scheme type is invalid.
sections/custom-liquid.liquid: Invalid schema: setting color_scheme type is invalid.
sections/email-signup-banner.liquid: Invalid schema: setting color_scheme type is invalid.
sections/featured-blog.liquid: Invalid schema: setting color_scheme type is invalid.
sections/main-password-footer.liquid: Invalid schema: setting color_scheme type is invalid.
sections/main-password-header.liquid: Invalid schema: setting color_scheme type is invalid.
sections/multicolumn.liquid: Invalid schema: setting color_scheme type is invalid.
sections/multirow.liquid: Invalid schema: setting color_scheme type is invalid.
sections/newsletter.liquid: Invalid schema: setting color_scheme type is invalid.
sections/page.liquid: Invalid schema: setting color_scheme type is invalid.
sections/rich-text.liquid: Invalid schema: setting color_scheme type is invalid.
sections/slideshow.liquid: Invalid block 'slide': setting color_scheme type is invalid.
sections/footer.liquid: Invalid schema: setting color_scheme type is invalid.
sections/header.liquid: Invalid schema: setting color_scheme type is invalid.
sections/image-banner.liquid: Invalid schema: setting color_scheme type is invalid.
sections/image-with-text.liquid: Invalid schema: setting color_scheme type is invalid.
sections/main-collection-banner.liquid: Invalid schema: setting color_scheme type is invalid.
sections/video.liquid: Invalid schema: setting color_scheme type is invalid.
sections/footer-group.json: Section type 'footer' does not refer to an existing section file
sections/header-group.json: Section type 'announcement-bar' does not refer to an existing section file
sections/header-group.json: Section type 'header' does not refer to an existing section file
templates/cart.json: Section type 'featured-collection' does not refer to an existing section file
templates/collection.json: Section type 'main-collection-banner' does not refer to an existing section file
templates/index.json: Section type 'image-banner' does not refer to an existing section file
templates/index.json: Section type 'rich-text' does not refer to an existing section file
templates/index.json: Section type 'featured-collection' does not refer to an existing section file
templates/index.json: Section type 'collage' does not refer to an existing section file
templates/index.json: Section type 'video' does not refer to an existing section file
templates/index.json: Section type 'multicolumn' does not refer to an existing section file
templates/page.contact.json: Section type 'contact-form' does not refer to an existing section file
templates/password.json: Section type 'email-signup-banner' does not refer to an existing section file
templates/product.json: Section type 'image-with-text' does not refer to an existing section file
templates/product.json: Section type 'multicolumn' does not refer to an existing section file
  • 22:33:16 ERROR  » update config/settings_schema.json:
  Section 3: setting with id="color_schemes" invalid setting type
Please open this URL in your browser:
http://127.0.0.1:9292
```

### Reproduction steps
Basically this article https://shopify.dev/docs/themes/getting-started/create
- installed shopify cli locally
- had the correct development environment
- had the local preview store up and running after using this command `shopify theme dev --store=https://mystore`
- changed the value of the title in the sections/main-404.liquid

### Operating System

Wins 10, WSL2

### Shopify CLI version (check your project's `package.json` if you're not sure)

3.45.3

### Node version (run `node -v` if you're not sure)

16.19.1

### What language and version are you using in your application?

Ruby 3.2.2

**Comment by yog18r** [comment]
FYI this appears to be a duplicate issue of #1912  and relates to Dawn releasing a feature before it is supported. There is a workaround in there to get you up and running. 
`shopify theme init --latest`

**Comment by anakinsolo** [comment]
> FYI this appears to be a duplicate issue of #1912 and relates to Dawn releasing a feature before it is supported. There is a workaround in there to get you up and running. `shopify theme init --latest`

thanks for your reply. My work around was to pull the Dawn theme from my demo shop using `shopify theme pull` and it seems to work
